src/tests/cptests/Makefile

```diff
@@ -38,7 +38,13 @@ clean:
 fuzz_parent_test_objects = $(foreach obj,$(notdir $(parent_test_objects)),fuzz-$(obj))
 fuzz_objects = $(foreach obj,$(parent_objs),fuzz-$(obj))
 
-fuzz: prepare-incdir fuzz.cc $(fuzz_parent_test_objects) $(fuzz_objects)
+# Create mconfig.h for fuzzing. because fuzz target not runned by top level make; we need to create mconfig.h manually.
+gen-mconfig:
+	$(MAKE) -C ../../../ mconfig
+	$(MAKE) -C ../../../build/tools/
+	../../../build/tools/mconfig-gen | tee includes/mconfig.h
```

Comment:
since this rule generates a file (`includes/mconfig.h`) why not make that the target rather than a phony target (`gen-mconfig`)? However, I also favour building mconfig.h in the usual place (`build/includes`) via the existing "make -C ../../../build all" and then just adding the `../../../build/includes` directory to the include path in the compilation targets below

Reply:
> since this rule generates a file (includes/mconfig.h) why not make that the target rather than a phony target (gen-mconfig)?

What do you mean? That is, how to make mconfig.h as "target"?

> However, I also favour building mconfig.h in the usual place (build/includes) via the existing "make -C ../../../build all" and then just adding the ../../../build/includes directory to the include path in the compilation targets below

Ok; i use it instead

Reply:
> What do you mean? That is, how to make mconfig.h as "target"?

I mean instead of `gen-mconfig:` you could have `includes/mconfig.h:`. Since this is a real file it means it doesn't get re-built if it doesn't need to. A phony target (`gen-mconfig`) isn't a real file and so gets rebuilt every time.

> Ok; i use it instead

:+1: 

you should probably have something like:

```
../../../build/includes/mconfig.h: ../../../mconfig
	$(MAKE) -C ../../../ mconfig
	$(MAKE) -C ../../../build all
```

Reply:
Ok; i fix it now

Reply:
We have a new problem.
@davmac314 I dont have any idea about this; Please check this out: https://github.com/mobin-2008/dinit/runs/8034167634

Reply:
I run `cat includes/mconfig.h` and everything looks good:
````
// This file is auto-generated by mconfig-gen.cc.

#ifndef DINIT_MCONFIG_H
#define DINIT_MCONFIG_H 1

// Defines

#define SUPPORT_CGROUPS 1

// Constants

constexpr static char DINIT_VERSION[] = "0.16.0pre";
constexpr static char SYSCONTROLSOCKET[] = "/run/dinitctl";
constexpr static char SBINDIR[] = "/sbin";
constexpr static char SHUTDOWN_PREFIX[] = "";

#endif // DINIT_MCONFIG_H
````

Reply:
Try with latest change on master.

Reply:
Fixed :+1: